crates/sui/src/keytool.rs

```diff
@@ -5,7 +5,16 @@ use bip32::DerivationPath;
 use clap::*;
 use fastcrypto::encoding::{decode_bytes_hex, Base64, Encoding, Hex};
 use fastcrypto::hash::HashFunction;
+use fastcrypto::hash::Sha256;
+use fastcrypto::secp256k1::recoverable::Secp256k1RecoverableSignature;
+use fastcrypto::secp256k1::recoverable::Secp256k1Sig;
+use fastcrypto::secp256k1::Secp256k1Signature;
 use fastcrypto::traits::KeyPair;
+use fastcrypto::traits::RecoverableSignature;
+use fastcrypto::traits::ToFromBytes;
+use fastcrypto::traits::VerifyingKey;
+use rusoto_core::Region;
```

Comment:
do u need this dep rusoto_core? why not use the Region from rusoto_kms? https://docs.rs/rusoto_kms/latest/rusoto_kms/struct.KmsClient.html